src/main/java/com/exasol/adapter/request/parser/TablesMetadataParser.java

```diff
@@ -77,35 +82,35 @@ private DataType getDataType(final JsonObject dataType) {
         final String typeName = dataType.getString("type").toUpperCase();
         final DataType type;
         switch (typeName) {
-        case "DECIMAL":
-            type = getDecimalDataType(dataType);
-            break;
-        case "DOUBLE":
-            type = getDoubleDataType();
-            break;
-        case "VARCHAR":
-            type = getVarcharDataType(dataType);
-            break;
-        case "CHAR":
-            type = getCharDataType(dataType);
-            break;
-        case "BOOLEAN":
-            type = getBooleanDataType();
-            break;
-        case "DATE":
-            type = getDateDataType();
-            break;
-        case "TIMESTAMP":
-            type = getTimestampDataType(dataType);
-            break;
-        case "INTERVAL":
-            type = getIntervalDataType(dataType);
-            break;
-        case "GEOMETRY":
-            type = getGeometryDataType(dataType);
-            break;
-        default:
-            throw new RequestParserException("Unsupported data type encountered: " + typeName);
+            case "DECIMAL":
```

Comment:
This is not consistent formatting with above another class with `switch`.

Please create a ticket for having unified formatter.

Reply:
Delegated to @AnastasiiaSergienko.